### PR TITLE
AT 00.01.01 | New item > Freestyle Project> Create Freestyle Project Using Valid Name

### PR DESCRIPTION
**Pull request by kristi-w**
Creating freestyle project with valid name

### CODE REVIEW

**Comment by kristi-w** [comment]
[TC 00.01.01 | New item > Freestyle Project> Create Freestyle Project Using Valid Name](https://trello.com/c/Lnb4Fkq3/71-tc-000101-new-item-freestyle-project-create-freestyle-project-using-valid-name)

File: src/test/java/CreateFreestyleProjectTest.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        getDriver().findElement(BUTTON_OK).click();
+        getDriver().findElement(By.xpath("//textarea[@name='description']")).sendKeys(description);
+        getDriver().findElement(BUTTON_SAVE).click();
+
+        Assert.assertEquals(getDriver().findElement(By.xpath("//div[@id='main-panel']/h1")).getText(),
+                "Project " + validFreestyleProjectName);
+        Assert.assertEquals(getDriver().findElement(By.xpath("//div[@id='description']/div[1]")).getText(),
+                description);
+    }
+
```
[the remark this answers is not in the document]
Comment (kristi-w) [reply src/test/java/CreateFreestyleProjectTest.java]: removed empty line

File: src/test/java/CreateFreestyleProjectTest.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import org.openqa.selenium.By;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import runner.BaseTest;
+
+public class CreateFreestyleProjectTest extends BaseTest {
+
+    private final String validFreestyleProjectName = "First project";
+    private final String description = "Description Text";
```
[the remark this answers is not in the document]
Comment (kristi-w) [reply src/test/java/CreateFreestyleProjectTest.java]: added 'static' to 2 variables

**Comment by kristi-w** [comment]
[AT 00.01.01 | New item > Freestyle Project> Create Freestyle Project Using Valid Name](https://trello.com/c/4WsvyN1a/407-at-000101-new-item-freestyle-project-create-freestyle-project-using-valid-name)

**Review by dmcalifornia — APPROVED** [review]

**Review by starnetsd — APPROVED** [review]

**Pull request merged**